__init__.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import requests
+
+from tests.constants import constants
+
+
+class ApiTestBase(unittest.TestCase):
+
+    def test_pass(self):
+        self.assertTrue(True)
+
+    def test_fail(self):
+        self.assertTrue(False)
+
+    def test_error(self):
+        raise RuntimeError('Test error!')
+
+
+if __name__ == '__main__':
```

Comment:
We will run our tests with `nose` test runner, please remove these lines of code from here